static/bidder-info/admatic.yaml

```diff
@@ -0,0 +1,22 @@
+endpoint: "http://pbs.admatic.com.tr/?host={{.Host}}"
+endpointCompression: gzip
+maintainer:
+  email: "abdulbaki.cam@admatic.com.tr"
+gvlVendorID: 1281

```

Comment:
@bakicam gvl id seems incorrect

```
curl https://vendor-list.consensu.org/v2/vendor-list.json | jq '.vendors."1281"'

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  430k  100  430k    0     0   607k      0 --:--:-- --:--:-- --:--:--  606k
null
```

Reply:
Hello, I can see it's defined here. https://vendor-list.consensu.org/v3/vendor-list.json